### PR TITLE
Fix Microsoft.Extensions.AuditReports output path

### PR DESCRIPTION
**Pull request by xakep139**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/extensions/pull/4945)

### CODE REVIEW

**Comment by xakep139** [comment]
@joperezr is my understanding correct, that it will land in 8.3?

**Comment by joperezr** [comment]
Correct, all changes going into main now, up until 1 week before the release, will be included in 8.3

**Review by joperezr — APPROVED** [review]
While not directly related to this, but would it be possible to also use this PR to finish up the README for this package? Right now there is basically no docs at all so the only way to consume this package is to know about it, and then to know how to read MSBuild in order to figure out what needs to be done in order to generate the reports.

**Review by mobratil — APPROVED** [review]

**Comment by xakep139** [comment]
> While not directly related to this, but would it be possible to also use this PR to finish up the README for this package? Right now there is basically no docs at all so the only way to consume this package is to know about it, and then to know how to read MSBuild in order to figure out what needs to be done in order to generate the reports.

Good point, will fill README

File: src/Shared/DiagnosticIds/DiagnosticIds.cs [hunk 1]
```diff
@@ -111,6 +111,12 @@ internal static class Metrics
         internal const string METGEN018 = nameof(METGEN018);
         internal const string METGEN019 = nameof(METGEN019);
     }
+
+    internal static class AuditReports
+    {
+        internal const string AUDREP000 = nameof(AUDREP000);
+        internal const string AUDREP001 = nameof(AUDREP001);
```
Comment (RussKie) [inline src/Shared/DiagnosticIds/DiagnosticIds.cs:118]: Do we need to document these in https://github.com/dotnet/extensions/blob/main/docs/list-of-diagnostics.md

Comment (xakep139) [reply src/Shared/DiagnosticIds/DiagnosticIds.cs:118]: Done both: https://aka.ms/dotnet-extensions-warnings/AUDREPGEN001

File: global.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "8.0.101"
+    "version": "8.0.200"
```
Comment (xakep139) [inline global.json:3]: @RussKie are we allowed to change that manually?

Comment (RussKie) [reply global.json:3]: yes, any time as required

**Review by dariusclay — APPROVED** [review]

**Pull request merged**